### DISCUSSION TITLE
Arctic code vault question

### DISCUSSION THREAD

**Discussion by Jme7** [community/community#22697] [New to GitHub]
<p>How often is the ACV updated with new repositories, and when will it check next? I know that <a href="https://github.community/t/earning-the-github-arctic-code-vault-badge/124219" class="inline-onebox">Earning The GitHub Arctic Code Vault Badge</a> said to go to the FAQ if you have questions, but that didn’t help.</p>

**Comment by mikerochip** [comment]
<p>Not sure if this is what you’re getting at OP, but honestly for me it feels pretty bad that I missed the window for this by a few days and that there are no current plans to do it again.</p>
<p>I think mostly what feels bad is that this badge is prominently displayed on people’s profiles so it feels a little bit like a haves vs have-nots situation with no possibility of earning it. A little bit like the Twitter verifies checkmark but not obviously not exactly the same thing.</p>
<p>I might even feel better just by not seeing it on a person’s profile unless I drill into a detailed view or something!</p>
<p>Totally acknowledge that I’m just projecting here but this is how it feels to me.</p>

**Comment by Jme7** [comment]
<p>Yea, I missed the window too. So did you say that they have no plans to do it again?</p>

**Comment by mikerochip** [comment, marked as answer]
<p>Not currently. It’s on the FAQ page:</p>
<blockquote>
<p>Are the repositories in the GitHub Arctic Code Vault at the Arctic World Archive ever updated?</p>
</blockquote>
<blockquote>
<p>We plan to evaluate the program, and the state of the art of archival technology, every five years. Depending on the results of each evaluation, we may then decide to take another snapshot of GitHub’s public code and archive it in cold storage.</p>
</blockquote>

**Comment by Jme7** [comment]
<p><a class="mention" href="/mikerochip">@mikerochip</a> Ok, thanks!</p>
